### ISSUE TITLE
Can't seem to fix two small issues.

### ISSUE THREAD

**Issue by sunscreem** [putyourlightson/craft-sherlock#6]
In my general.php I have:

```
   'cooldownDuration'                => 600,
    'invalidLoginWindowDuration'      => 7200,
```

But I'm still getting this:

![image](https://user-images.githubusercontent.com/7523595/59510613-0ae7e700-8eac-11e9-9f83-b3cb560a9e1e.png)

I'm probably being dumb.

**Comment by putyourlightson** [comment]
Thanks, we're looking into this and will release an update with a fix soon.

**Comment by putyourlightson** [comment]
Fixed in version 2.1.1.
